cpp/daal/src/algorithms/dtrees/gbt/regression/gbt_regression_tree_impl.h

```diff
@@ -312,11 +312,20 @@ class TreeTableConnector
 
     template <CpuType cpu>
     void convertToGbtDecisionTree(algorithmFPType ** binValues, const size_t nNodes, const size_t maxLevel, gbt::internal::GbtDecisionTree * tree,
-                                  double * impVals, int * nNodeSamplesVals, const algorithmFPType initialF, const training::Parameter & par)
+                                  double * impVals, int * nNodeSamplesVals, const algorithmFPType initialF, const training::Parameter & par, services::Status * status)
```

Comment:
As I remember we should remove pointer to Status from the library and use reference instead of.

Reply:
Yes, please use reference instead of pointer to Status - it is unsafe

Reply:
Fixed.